bibformat/format_elements/bfe_INSPIRE_enhanced_marcxml.py

```diff
@@ -211,6 +232,27 @@ def format_element(bfo, oai=0):
                     if len(ids) == 1:
                         subfields.append(('z', '%i' % ids[0]))
 
+    for field in record_get_field_instances(record, '110'):
+        subfields = field_get_subfield_instances(field)
+        subfield_dict = dict(subfields)
+        if is_institution:
+            # We try to resolve obsolete ICNs
+            if 'w' in subfield_dict:
```

Comment:
Shouldn't this guy be `x`? https://twiki.cern.ch/twiki/bin/view/Inspire/DevelopmentRecordMarkupInstitutions#Field_Mapping_final